scripts/start-selenium-env

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Make sure we always run from the root
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$SCRIPTS_DIR/.."
+
+#########
+# USAGE #
+#########
+
+function title() {
+    echo "Launch docker based Selenium test infrastructure (selenium and nginx containers)"
```

Comment:
We used to have `with-selenium-standalone` which ran a given command with the environment up, meaning you didn't have to worry about shutting anything down. Do you think it'd make sense to have the same here?

Reply:
Yes, but I will do that in a separate PR. 👍